src/EventListener/ResizeFormListener.php

```diff
@@ -108,7 +111,10 @@ public function preSubmit(FormEvent $event): void
         }
 
         // Remove all empty rows except for the prototype row
-        // Type cast to string, because Symfony form can returns integer keys
+        // Type cast to string, because Symfony form can return integer keys
+        /**
+         * @psalm-suppress RedundantCastGivenDocblockType
+         */
```

Comment:
This will be corrected in SF 6.2 with
https://github.com/symfony/symfony/pull/46813

Might be interesting to add a comment